src/main/java/org/broadinstitute/hellbender/utils/variant/writers/HomRefBlock.java

```diff
@@ -154,7 +156,9 @@ public void add(final int pos, final int newEnd, final Genotype genotype) {
         }
 
         end = newEnd;
-        DPs.add(Math.max(genotype.getDP(), 0)); // DP must be >= 0
```

Comment:
Wonder when could it be less than 0? No changes needed.

Reply:
Since it's a primitive int it needs a value and it's -1 if DP is missing.